infrasim/model.py

```diff
@@ -923,8 +923,8 @@ def handle_parms(self):
 
 class CBMC(Task):
 
-    VBMC_TEMP_CONF = "/usr/local/etc/infrasim/conf/vbmc.conf"
-    VBMC_CONF = "/etc/infrasim/vbmc.conf"
+    VBMC_TEMP_CONF = TEMPLATE_ROOT + "template/vbmc.conf"
+    VBMC_CONF = TEMPLATE_ROOT + "etc/vbmc.conf"
```

Comment:
template file can still be obtained from /user/local/infrasim/template if we decide not using template files in ~/.infrasim